roles/kubernetes-apps/metallb/README.md

```diff
@@ -3,13 +3,13 @@
 MetalLB hooks into your Kubernetes cluster, and provides a network load-balancer implementation.
 In short, it allows you to create Kubernetes services of type "LoadBalancer" in clusters that
 don't run on a cloud provider, and thus cannot simply hook into paid products to provide load-balancers.
-This addon aims to automate [this](https://metallb.universe.tf/concepts/layer2/).
-It deploys MetalLB into Kubernetes and sets up a layer 2 load-balancer.
+This addon aims to automate [this](https://metallb.universe.tf/concepts/layer2/) and/or [this][https://metallb.universe.tf/concepts/bgp/].
```

Comment:
There will be 2 `this` on README.md and that would make a confusion.
How about changing the first `this` to `MetalLB in layer 2 mode` and later `this` to `MetalLB in BGP mode` as the same as the links?

Reply:
Yep, fair point, "this" wasn't great adding another was bad.
I've changed them as per your suggestion.